lib/typinggame_server/interactors/players_rooms/create_players_rooms.rb

```diff
@@ -0,0 +1,22 @@
+require 'hanami/interactor'
+
+module Interactors
+  module PlayersRooms
+    class CreatePlayersRooms
```

Comment:
This creates a player room, singular. Probably worth updating the name the reflect that.

Reply:
@jgayfer updated.